api/src/main/java/ca/bc/gov/educ/api/edx/validator/EdxActivationCodeSagaDataPayLoadValidator.java

```diff
@@ -28,6 +30,18 @@ public List<FieldError> validateEdxActivationCodeSagaDataPayload(EdxUserActivati
     return apiValidationErrors;
   }
 
+  public List<FieldError> validateEdxActivationCodeRelinkSagaDataPayload(EdxUserActivationRelinkSagaData edxUserActivationRelinkSagaData) {
+    final List<FieldError> apiValidationErrors = new ArrayList<>();
+    if(StringUtils.isBlank(edxUserActivationRelinkSagaData.getEdxUserID())){
+      apiValidationErrors.add(createFieldError("edxUserID", edxUserActivationRelinkSagaData.getEdxUserID(), "EDX User ID must be provided for re-link"));
+    }
+    if(StringUtils.isBlank(edxUserActivationRelinkSagaData.getEdxUserSchoolID())){
+      apiValidationErrors.add(createFieldError("edxUserSchoolID", edxUserActivationRelinkSagaData.getEdxUserSchoolID(), "EDX User School ID must be provided for re-link"));
+    }
+    apiValidationErrors.addAll(validateEdxActivationCodeSagaDataPayload(edxUserActivationRelinkSagaData));
+    return apiValidationErrors;
```

Comment:
The endpoint has @Validated and the Struct has Not null constraints. I think we don't need this.